src/components/graph/util/center-graph-button.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * Copyright (c) 2021, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+import CenterFocusIcon from '@material-ui/icons/CenterFocusStrong';
+import React, { useCallback } from 'react';
+import { ControlButton, useZoomPanHelper } from 'react-flow-renderer';
+
+const CenterGraphButton = ({ selectedNode }) => {
+    const { setCenter } = useZoomPanHelper();
+
+    const focusNode = useCallback(() => {
```

Comment:
If we need to center over the selectedNode without using this ControlButton, for example when creating node, or selecting another node in a list in another component or anyway, how could we call this focusNode function.
Maybe we should externalized it in a utilitary js file and call it here, change which allow to call this function directly from network-modification-tree or others component if needed.
I see that coming...

Reply:
Ok